### PR TITLE
Prevents Syndie Nukes from being moved without the NAD

### PR DESCRIPTION
**Pull request by Coolrune206**
## What Does This PR Do

Adds an additional variable to Syndie nukes which prevents the first step of tool-based unanchoring from being done until the NAD is used to unbolt the nuke. Once the nuke is unbolted, this variable is set to false which means it can be un-anchored by tools so crew can still try to unanchor it and space it if it's actually planted on station.
## Why It's Good For The Game

The nukies game-mode is intended to be crew vs nukies, actually fighting for defense of the station. Instead, some players have figured out that they can entirely circumvent the gamemode by easily breaking into the nukie shuttle (it's poorly defended), and it's genuinely unfair to say that one of the players picked for nukie should have to sit behind on the shuttle and not enjoy the fun of the round just because some people want to cheese the roundtype. This prevents that cheese from being effective (people can still break into the shuttle to steal spare nukie gear), so people can actually enjoy the roundtype itself. This PR was requested by MattTheFicus.
## Testing

Ensured I could not weld the syndie nuke initially, popped in an NAD, spent 2 minutes figuring out what the variable for the nuke code was, unbolted the nuke, rebolted it, yote the NAD, tried to weld it, succeeded. Confirmed the crew's nuke could be welded normally regardless.
## Changelog
:cl:
tweak: Syndicate Nukes can now only be unbolted with the NAD on the first unbolting. After that, they can be unbolted by tools.
/:cl:

### CODE REVIEW

**Comment by matttheficus** [comment]
I said I wanted this months ago after seeing the Nuke get twice in a row.

It has now happened a third time.

I Ded Plz Nerf this all you want, I approve.

**Comment by warriorstar-orion** [comment]
RIP Vacationuke.
2023_01_08__01_32_10__Paradise Station 13

**Comment by Carthusia** [comment]
I thought this was alreadya thing, also that vacation nuke is just amazing

File: code/game/gamemodes/nuclear/nuclearbomb.dm [hunk 1]
```diff
@@ -33,6 +33,7 @@ GLOBAL_VAR(bomb_set)
 	var/obj/item/nuke_core/plutonium/core = null
 	var/lastentered
 	var/is_syndicate = FALSE
+	var/requires_NAD_to_unbolt = FALSE // if this is true you cannot unbolt the NAD with tools, only the NAD
```
Comment (farie82) [inline code/game/gamemodes/nuclear/nuclearbomb.dm:36]:
```suggestion
	/// If this is true you cannot unbolt the NAD with tools, only the NAD
	var/requires_NAD_to_unbolt = FALSE
```

File: code/game/gamemodes/nuclear/nuclearbomb.dm [hunk 2]
```diff
@@ -227,6 +229,9 @@ GLOBAL_VAR(bomb_set)
 	. = TRUE
 	if(!I.tool_use_check(user, 0))
 		return
+	if(requires_NAD_to_unbolt == TRUE)
```
Comment (farie82) [inline code/game/gamemodes/nuclear/nuclearbomb.dm:232]:
```suggestion
	if(requires_NAD_to_unbolt)
```

**Review by farie82 — APPROVED** [review]

**Pull request merged**